### DISCUSSION TITLE
Elección competencia

### DISCUSSION THREAD

**Discussion by diegorg07** [rramosp/ai4eng.v1#269] [00 - General]
Buenas días. 

Estuvimos mi equipo y yo buscando competencias que nos puedan ser útiles para la realización del proyecto, y teníamos la duda de si esta es posible y cumple con los requisitos.

https://www.kaggle.com/competitions/playground-series-s3e20/overview/description

Muchas gracias!

**Comment by Felipe-RA** [comment]
Saludos @diegorg07 

El dataset de vuestra competencia cumple que:

- Tiene más de 5000 instancias
- Tiene más de 30 columnas
- Parece cumplir con el requisito de tener 5% de datos faltantes en al menos tres columnas (a primera vista)

Recuerden que pueden simular el 10% de las variables categoricas, al igual que pueden simular más datos faltantes si no llegan a contabilizar al menos un 5%

En resumen, si cumple pero necesitan tener en cuenta lo que les mencioné arriba.

¡Hasta pronto, cualquier duda me avisan!
